abr_control/controllers/osc.py

```diff
@@ -43,7 +43,8 @@ class OSC(controller.Controller):
         task-space integrated error term
     """
     def __init__(self, robot_config, kp=1, kv=None, ki=0, vmax=0.5,
-                 null_control=True, use_g=True, use_C=False, use_dJ=False):
+                 null_control=True, use_g=True, use_C=False, use_dJ=False,
+                 integrated_error=None):
```

Comment:
have to be able to pass in an integrated error if doing continued learning between runs

Reply:
i think we can leave this out and create a wrapper class for doing continued learning on the integrated error term

Reply:
What's the reasoning for having an entire subclass to remove one parameter from the init?

Reply:
to keep the 99% use case code clean